keras_cv/models/resnet_v2.py

```diff
@@ -464,6 +484,7 @@ def ResNet18V2(
         stackwise_filters=MODEL_CONFIGS["ResNet18V2"]["stackwise_filters"],
         stackwise_blocks=MODEL_CONFIGS["ResNet18V2"]["stackwise_blocks"],
         stackwise_strides=MODEL_CONFIGS["ResNet18V2"]["stackwise_strides"],
+        stackwise_dilations=MODEL_CONFIGS["ResNet50V2"]["stackwise_dilations"],
```

Comment:
It looks like these are not defined in this file (and in this case it's using the config from the wrong ResNet)

Reply:
good catch. removed.